.github/workflows/continuous-integration.yml

```diff
@@ -379,18 +378,16 @@ jobs:
             .cache/yarn
             node_modules
 
-      - name: Set IS_MASTER_BUILD
-        run: |
-          if [[ ${{ github.ref }} == 'refs/heads/master' ]]; then
-              echo "IS_MASTER_BUILD=true" >> $GITHUB_ENV
-          else
-              echo "IS_MASTER_BUILD=false" >> $GITHUB_ENV
-          fi
-
       - name: List changed files
         id: changed-files
+        if: ${{ github.ref != 'refs/heads/master' }}
         run: echo ::set-output name=all_changed_files::$(git diff --name-only origin/master...)
 
+      - name: List changed files - master
```

Comment:
The same thing I mentioned in this [comment](https://github.com/department-of-veterans-affairs/vets-website/pull/20413#r819194274) applies here also. If we could combine the two `List changed files` steps into one, it would prevent the need for an extra `Set TESTS variable` step